socorro/unittest/cron/jobs/test_archivescraper.py

```diff
@@ -0,0 +1,313 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+from jinja2 import Environment
+import psycopg2
+import pytest
+import requests_mock
+
+from socorro.cron.crontabber_app import CronTabberApp
+from socorro.cron.jobs.archivescraper import ArchiveScraperCronApp
+from socorro.unittest.cron.crontabber_tests_base import get_config_manager
+
+
+HOST = 'http://archive.example.com'
+
+INDEX_PAGE_TEMPLATE = """
```

Comment:
This matches the structure that archive.mozilla.org has for index pages with one minor caveat in that it doesn't have the `..` link. I thought about adding that, but then was like, "meh." We can do it in a future PR.